views/model/model.blade.php

```diff
@@ -5,15 +5,21 @@
 namespace {{ $config->namespaces->model }};
 
 use Illuminate\Database\Eloquent\Model;
-@if($config->options->softDelete) {{ 'use Illuminate\Database\Eloquent\SoftDeletes;' }}@endif
-@if($config->options->tests or $config->options->factory) {{ 'use Illuminate\Database\Eloquent\Factories\HasFactory;' }}@endif
-
-@if(isset($swaggerDocs)){!! $swaggerDocs  !!}@endif
+@if($config->options->softDelete)
+{{ 'use Illuminate\Database\Eloquent\SoftDeletes;' }}
+@endif
+@if($config->options->tests or $config->options->factory)
+{{ 'use Illuminate\Database\Eloquent\Factories\HasFactory;' }}
+@endif
+@if(isset($swaggerDocs))
+{!! $swaggerDocs  !!}
+@else@nls(2)@endif
 class {{ $config->modelNames->name }} extends Model
 {
-@if($config->options->softDelete) {{ infy_tab().'use SoftDeletes;' }}@endif
-@if($config->options->tests or $config->options->factory){{ infy_tab().'use HasFactory;' }}@endif
-    public $table = '{{ $config->tableName }}';
+@if($config->options->softDelete && ($config->options->tests or $config->options->factory))@tab()use SoftDeletes, HasFactory;@nls(2)
```

Comment:
Combine `SoftDeletes` and `HasFactory` in a single use statements if there are both set to true in config. and replace incorrect space with a regular tab.